src/components/DialogOfContinuation.jsx

```diff
@@ -0,0 +1,43 @@
+/*
+Dialog Of Continuation
+-----------------
+ASM has a special problem: sometimes, when a user is working on a document for
+too long, their login session dies. Fortunately, we have an Emergency Save
+system that saves the user's work and lets them resume. This dialog informs
+them that their progress has been restored.
+ */
+
+import React from 'react';
+import { clearEmergencySave } from '../ducks/emergency-save';
+
+const DialogOfContinuation = ({ dispatch, onClose }) => {
+  return (
+    <div className="classification-prompt">
+      <h2>Resuming Work</h2>
+      <span>
+        We detected that you encountered a problem recently, and have restored
+        your work in progress. You can continue your saved work, or start with
+        a new manuscript. Thank you for your patience.
+      </span>
+      <div>
+        <button
+          className="button"
+          onClick={(e) => {
+            dispatch(clearEmergencySave());
+            location.reload();
+          }}
+        >
+          New Manuscript
+        </button>
+        <button
+          className="button"
+          onClick={(e) => { onClose && onClose(e); }}
```

Comment:
I think after the annotation is loaded when clicking this button this should also `clearEmergencySave()`, otherwise the following might happen:

1) User makes some annotations and saves classification successfully
2) User makes some annotations, saves classification and it fails, emergency save kicks in
3) User reloads page and loads emergency classification
4) User makes some annotations and saves classification successfully
5) User reloads the page, but the emergency classification is again loaded from point number 3 above (losing any annotations at point 4)

Reply:
This is a fair point.

Previously, I _purposely_ did not clear the emergency save _until_ the Classification was submitted, because there was a scenario where:
1. a user did work
2. user was logged out, then reloaded the page,
3. user returned to the page (but was logged out), quite possibly resuming their work,
4. then went "whoops! I need to login again", did so, and...
5. returned to the page with the emergency save data removed.

With the recent update for this branch (via PR #254), a user _has_ to be logged in to resume the work they did, so it's now fair to clear the emergency save when that happens. The manual Save Progress then takes the responsibility for maintaining user progress.